_overviews/cheatsheets/index.md

```diff
@@ -353,6 +353,223 @@ Some(3) match {
     <tr>
       <td><code class="highlighter-rouge">x: String</code></td>
       <td>ascription (compile time)</td>
+      <td> </td>
+    </tr>
+    <tr>
+      <td><span id="options" class="h2">options</span></td>
+      <td> </td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>Some(42)</code></pre></td>
+      <td>Construct a non empty optional value</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>None</code></pre></td>
+      <td>The singleton empty optional value</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>Option(null) == None
+Option(obj.unsafeMethod)</code></pre></td>
+      <td>Null-safe optional value factory</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>val optStr: Option[String] = None</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>val optStr = Option.empty[String]</code></pre></td>
+      <td>Explicit type for empty optional value.<br /> Factory for empty optional value.</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>val name: Option[String] =
+  request.getParameter("name")
+val upper = name.map {
+  _.trim
+}
+.filter {
+  _.length != 0
+}
+.map {
+  _.toUpperCase
+}
+println(upper.getOrElse(""))
+</code></pre></td>
+      <td>Pipeline style</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>val upper = for {
+  name <- request.getParameter("name")
+  trimmed <- Some(name.trim)
+    if trimmed.length != 0
+  upper <- Some(trimmed.toUpperCase)
+} yield upper
+println(upper.getOrElse(""))</code></pre></td>
+      <td>for-comprehension syntax</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.map(f(_))</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) =&gt; Some(f(x))
+  case None    =&gt; None
+}</code></pre></td>
+      <td>Apply a function on the optional value</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.flatMap(f(_))</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) =&gt; f(x)
+  case None    =&gt; None
+}</code></pre></td>
+      <td>Same as map but function must return an optional value</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.foreach(f(_))</code></pre>
+      <em><strong>same as</strong></em>
+      <pre class="highlight"><code>option match {
+  case Some(x) =&gt; f(x)
+  case None    =&gt; ()
+}</code></pre></td>
+      <td>Apply a procedure on optional value</td>
+    </tr>
+    <tr>
+      <td><pre class="highlight"><code>option.fold(y)f(_)</code></pre>
```

Comment:
that's not going to compile
you need to use option.fold(y)(f(_)) or option.fold(y)(f)
i persanaly prefer the last one, maybe just better drop (_) for every samples with f(_) what you use, cuz it's lead to such kind of misstyping

Reply:
Thanks, again.  

I think `f(_)` is fine.  The problem was I was just doing too many things at once, and also the HTML markup is distracting.  Too bad we can't have better Markdown tables.